roles/kokonet-setup/handlers/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- name: restart network
+  service:
+    name: network
+    state: restarted
+  listen: 'restart net'
```

Comment:
can we make this more explicit, like 'restart network service'

Reply:
fixed.